File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 3/IVisitYukio.cs
namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_3
{
	internal class IVisitYukio : StoryChoice
	{
		internal IVisitYukio() : base(StoryChoice.GetLocalizedString("ch3IVisitYukio"), 3086)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch3TalkedWithYukio = true;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_3/IVisitYukio.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_3;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;

/**
 * Created by christopherraleigh on 2016-05-12.
 */
public class IVisitYukio extends StoryChoice {
    public IVisitYukio() {
        super(R.string.ch3IVisitYukio, 3086);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh3TalkedWithYukio(true);
        super.select();
    }
}
